pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -535,6 +535,12 @@ def _set_parameters(self):
         eps_c_init_av = pybamm.xyz_average(
             self.epsilon_s * pybamm.r_average(self.c_init)
         )
+        # if self.options['open-circuit potential'] == 'Plett':
+        self.K = pybamm.Parameter(f"{pref}{Domain} particle hysteresis decay rate")
+        self.K_x = pybamm.Parameter(
+            f"{pref}{Domain} particle hysteresis switching factor"
+        )
+        self.h_init = pybamm.Scalar(0)
```

Comment:
I added these as default parameters. Not sure if I should try to create them only when specific model options have been turned on. What do you reccomend?

Reply:
They will only appear in the expression tree if these model options are turned on, so it's fine to add them here.